src/bin/inx-chronicle/main.rs

```diff
@@ -136,6 +140,7 @@ async fn main() -> eyre::Result<()> {
         res = process::interrupt_or_terminate() => {
             if let Err(err) = res {
                 tracing::error!("subscribing to OS interrupt signals failed with error: {err}; aborting");
+                exit_code = Err(err);
```

Comment:
the only thing that annoys me a tiny bit here (because that should never happen :trollface: ) is that we would overwrite a previous error which would then maybe mislead the user about the root cause for the error.

Reply:
Sure, but let's be honest, if you can't subscribe to interrupts you have bigger problems.

Reply:
yeah